sw/device/tests/BUILD

```diff
@@ -6279,3 +6279,24 @@ opentitan_test(
         "//sw/device/lib/testing/test_framework:ottf_utils",
     ],
 )
+
+opentitan_test(
+    name = "aes_interrupt_encryption_test",
+    srcs = ["aes_interrupt_encryption_test.c"],
+    exec_env = {
+        #         "//hw/top_earlgrey:sim_verilator": None,
+        "//hw/top_earlgrey:fpga_cw340_sival": None,
+        #         "//hw/top_earlgrey:fpga_cw340_rom_with_fake_keys": None,
```

Comment:
Please clean up the unsupported execution environments. Though perhaps you'd be able to add back verilator, etc?

Reply:
Done!